### ISSUE TITLE
Install udev rules to `/usr/lib/udev`

### ISSUE THREAD

**Issue by navnavnav** [bloombloombloom/Bloom#64]
Currently, Bloom's Debian, RPM and Pacman packages install the udev rules in `/etc/udev`, but this directory is meant for overriding rules. Default rules should be placed in `/usr/lib/udev`.

This is a small change - will be included in v0.12.0.

**Comment by navnavnav** [comment]
Quick update on this:

I've done this as part of the [build-tidying](https://github.com/bloombloombloom/Bloom/tree/build-tidying) branch.

The RPM package now installs Bloom's udev rules in `/usr/lib/udev/rules.d`. However, after trying to install the rules in the same place on a Debian-based system (Ubuntu 18.04), I discovered that rules in that location are not even loaded. Debian-based systems expect them to be installed in `/lib/udev/rules.d`, so that's where I've configured the DEB package to install them.

And it looks like, since Ubuntu 20.04, `/lib` is symlinked to `/usr/lib`, so I could have avoided having to install the rules in `/lib/udev/rules.d`, if I didn't package for Ubuntu 18.04. Oh well.
